mirgecom/viscous.py

```diff
@@ -342,6 +344,88 @@ def viscous_facial_flux_central(dcoll, state_pair, grad_cv_pair, grad_t_pair,
     return num_flux_central(f_int, f_ext)@normal
 
 
+def viscous_facial_flux_harmonic(dcoll, state_pair, grad_cv_pair, grad_t_pair,
+                                 gas_model=None):
```

Comment:
This inadvertently leaked into #868 , i'll move it out of there and let you add it here.